client/src/modules/templates/bhGridLoadingIndicator.tmpl.html

```diff
@@ -17,7 +17,7 @@
   <!-- indicate that some error occurred -->
   <div class="msg" ng-show="$ctrl.errorState">
     <span class="text-danger">
-      <span class="fa fa-warning"></span> <span translate>ERRORS.UNKNOWN</span>
+      <span class="fa fa-warning"></span> <span translate>{{ $ctrl.errorStateMessage || 'ERRORS.UNKNOWN' }}</span>
```

Comment:
In the future, this operation would be best done in your `$onInit` method.  It would allow you to easily use one-time binding (`::`), as well as clarify that the message will never change after the first initialization.